crates/store/re_types/definitions/rerun/archetypes/series_point.fbs

```diff
@@ -29,13 +28,6 @@ table SeriesPoint (
     /// Used in the legend.
     name: rerun.components.Name ("attr.rerun.component_optional", nullable, order: 3000);
 
-    /// Which point series are visible.
-    ///
-    /// If not set, all point series on this entity are visible.
-    /// Unlike with the regular visibility property of the entire entity, any series that is hidden
-    /// via this property will still be visible in the legend.
-    visible_series: [rerun.components.SeriesVisible] ("attr.rerun.component_optional", nullable, order: 3100);
```

Comment:
Why remove this from the deprecated type?

Reply:
this was newly added and shouldn't be used from the old type